### ISSUE TITLE
MacOS not supported

### ISSUE THREAD

**Issue by ramonemiliani93** [reloadware/reloadium#156]
## Describe the bug*
Unable to debug with reloadium on MacOS

## Desktop or remote (please complete the following information):**
 - OS: MacOS
 - OS version: Ventura 13.1
 - M1 chip: no
 - Reloadium package version: 1.2.0
 - PyCharm plugin version: 1.2.0
 - Editor: PyCharm
 - Python Version: 3.9.13
 - Python Architecture: 64bit
 - Run mode: Debug

## Additional context
```
ImportError: dlopen(/Users/ramon/.reloadium/package/3.9/reloadium/corium.cpython-39-darwin.so, 0x0002): tried: '/Users/ramon/.reloadium/package/3.9/reloadium/corium.cpython-39-darwin.so' (mach-o file, but is an incompatible architecture (have 'arm64', need 'x86_64h' or 'x86_64')), '/System/Volumes/Preboot/Cryptexes/OS/Users/ramon/.reloadium/package/3.9/reloadium/corium.cpython-39-darwin.so' (no such file), '/Users/ramon/.reloadium/package/3.9/reloadium/corium.cpython-39-darwin.so' (mach-o file, but is an incompatible architecture (have 'arm64', need 'x86_64h' or 'x86_64'))
```
MacBook Pro 2.9 GHz 6-Core Intel Core i9

**Comment by dkrystki** [comment]
If you run this command `sysctl -n machdep.cpu.brand_string` what is the output?

**Comment by ramonemiliani93** [comment]
Hi @dkrystki sorry for the late reply! The output is: `Intel(R) Core(TM) i9-8950HK CPU @ 2.90GHz`

**Comment by dkrystki** [comment]
Fixed in Reloadium 1.3.0 and PyCharm plugin 1.3.0
